### PR TITLE
NZZO, NTTT, NSFA, NZCM and internal NZZC amendments.

### PR DESCRIPTION
**Pull request by dirtyformal**
Hey team,

This is the first update to the new GeoJSON format so far, so hopefully it's all honky-dory. 

- Slight changes to the bounds of our already existing internal sectors, which interface with NZZO.
- NZCM date-line fix - #415
- Change of Tahiti/Auckland 'Oceanic' to 'Radio'
- Addition of an event-only Auckland Radio East callsign
- Addition of an event-only internal sector callsign (NZAA-R_CTR)
- Slight change in NZCM's name.

Tested with the latest 2112 GeoJSON bounds, and I'm happy. Any issues, just sing out!

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -18069,16 +18069,18 @@ NFFF|Nadi Oceanic||NFFF
 NFFN|Nadi||NFFN
 NVVV|Port Vila||NVVV
 NSTU|Pago Pago||NZZO
-NTTT|Tahiti Oceanic||NTTT
+NTTT|Tahiti Radio||NTTT
```
Comment (Adrian2k) [inline VATSpy.dat:18072]: We aim to only have the FIR names here when possible and as this is the only FIR that starts with "NT" I would suggest edit line 120 to include the "Radio" instead, like this:

> French Polynesia|NT|Radio

And then only have "Tahiti" as the FIR name.

File: VATSpy.dat [hunk 2]
```diff
@@ -18069,16 +18069,18 @@ NFFF|Nadi Oceanic||NFFF
...
 NZAA|Auckland||NZAA
+NZAA|Auckland|NZAA-R|NZAA
 NZCB|Christchurch (Bay)|NZCH-B|NZCB
 NZCT|Christchurch (Taranaki)|NZCH-T|NZCT
 NZCK|Christchurch (Kaikoura)|NZCH-K|NZCK
 NZCS|Christchurch (South)|NZCH-S|NZCS
-NZOH|Ohakea ||NZOH
-NZCM|McMurdo||NZCM
+NZOH|Ohakea||NZOH
+NZCM|McMurdo (Mac)||NZCM
```
Comment (Adrian2k) [inline VATSpy.dat:18083]: Out of curiosity, why has (Mac) been included here? As far as I can see, there is only one FIR named McMurdo here, no subsectors like Christchurch

**Review by Adrian2k — COMMENTED** [review]
Niels will have to review the geojson, but looked at the VATspy.dat changes.

File: VATSpy.dat (hunk 1 above)
Comment (dirtyformal) [reply VATSpy.dat:18072]: That makes sense, however we intend to add NTTT specific sector splits in the future, so this wouldn't work. This is the preferred solution.

File: VATSpy.dat (hunk 2 above)
Comment (dirtyformal) [reply VATSpy.dat:18083]: The item in parenthesis is typically that station's callsign. For example, Christchurch Control (Taranaki)'s RTF ident is Taranaki/Naki. In this case, NZCM's RTF ident is Mac. Roughly similar to how VATPAC handle theirs.

**Review by NelisV — APPROVED** [review]

**Review by Adrian2k — APPROVED** [review]

**Pull request merged**